src/main/resources/com/tresys/nitf/xsd/nitf.dfdl.xsd

```diff
@@ -480,8 +486,26 @@ The message root is 'NITF'.
                                 <xs:element name="Corner" minOccurs="4" maxOccurs="4" dfdl:occursCountKind="fixed">
                                   <xs:complexType>
                                     <xs:sequence>
-                                      <xs:element name="Latitude" type="nitf:BCS-A" dfdl:length="7" />
-                                      <xs:element name="Longitude" type="nitf:BCS-A" dfdl:length="8" />
+                                      <xs:element name="Latitude">
```

Comment:
I definitely like this change, but a few of the existing tests will need to be updated to expect the new Degrees/Minutes/Seconds/NorthSouth. Make sure to run sbt test to make sure they've all be fixed.